http/README.md

```diff
@@ -0,0 +1,153 @@
+# HTTP
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+| --------------- | ------------- | ------ | --------------- |
+| 1A              | Working Draft | Active | r0, 2023-01-23  |
+
+Authors: [@marten-seemann], [@MarcoPolo]
+
+Interest Group: [@lidel], [@thomaseizinger]
+
+[@marten-seemann]: https://github.com/marten-seemann
+[@MarcoPolo]: https://github.com/MarcoPolo
+[@lidel]: https://github.com/lidel
+[@thomaseizinger]: https://github.com/thomaseizinger
+
+## Introduction
+
+This document defines how libp2p nodes can offer and use an HTTP transport alongside their other transports to support application protocols with HTTP semantics. This allows a wider variety of nodes to participate in the libp2p network, for example:
```

Comment:
This will be my last comment. Expending effort on using HTTP is a signal that moves the center of gravity of the project — verbiage of scope or bindingness does not change this. The desire to make HTTP requests is best met by the programmer using an HTTP client library. You have not answered how a browser shall successfully make an HTTPS request to a peer, given that non-cloud peers will not be able to present valid X.509 certificates.